checks/evaluation/fuzzing.go

```diff
@@ -44,8 +48,8 @@ func Fuzzing(name string, dl checker.DetailLogger,
 				dl.Info(&msg)
 			}
```

Comment:
rename FuzzNameUserDefinedFunc to something like `FuzzerGoBuiltIn`, `FuzzerJavaBuiltIn`, etc